raiden-network/light-client
# Fixes navigation preserving after pressing connect

Fixes #1395 

**Short description**
Introduces lazy routes that evaluate after pressing the connect button.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Navigate to the channels page for a token page
2. Hit refresh
3. Press connect
4. See the channels page.
